src/auto-evo/AutoEvoConfiguration.cs

```diff
@@ -0,0 +1,41 @@
+using Newtonsoft.Json;
+
+public class AutoEvoConfiguration : IRegistryType
+{
+    [JsonProperty]
+    public int MutationsPerSpecies { get; private set; }
+
+    [JsonProperty]
+    public bool AllowNoMutation { get; private set; }
+
+    [JsonProperty]
+    public int MoveAttemptsPerSpecies { get; private set; }
+
+    [JsonProperty]
+    public bool AllowNoMigration { get; private set; }
+
+    /// <summary>
+    /// Unused
+    /// </summary>
+    /// <value>The name of the internal.</value>
```

Comment:
What even is this comment. Unused is fine to keep but it should be intended 2 spaces.